### PR TITLE
enh(UI): Handle close ticket on resources table

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This handles close ticket on resources table

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [x] master

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/www/widgets/src/centreon-widget-resourcestable/src/Listing/translatedLabels.ts [hunk 1]
```diff
@@ -83,3 +85,9 @@ export const labelTicketOpenTime = 'Opened on';
 export const labelSubject = 'Subject';
 export const labelOpenTime = 'Open time';
 export const labelCreateticket = 'Create a ticket';
+export const labelCloseATicket = 'Close a ticket';
+export const labelConfirm = 'Confirm';
+export const labelTicketWillBeClosedInTheProvider =
+  'The ticket will be closed on the selected ticket provider';
+export const labelTicketCreated = 'Ticket created';
+export const labelTicketClosed = 'Ticket closed';
```
Comment (Med1233) [inline centreon/www/widgets/src/centreon-widget-resourcestable/src/Listing/translatedLabels.ts:93]: Does the translation for these words already exist?

File: centreon/www/widgets/src/centreon-widget-resourcestable/src/spec/ResourcesTable.cypress.spec.tsx [hunk 2]
```diff
@@ -502,6 +523,97 @@ describe('Open tickets', () => {
...
+            rule_id: '1'
+          }
+        })
+      );
+    });
+
+    cy.contains(labelTicketClosed).should('be.visible');
+
+    cy.makeSnapshot();
+  });
```
Comment (Med1233) [inline centreon/www/widgets/src/centreon-widget-resourcestable/src/spec/ResourcesTable.cypress.spec.tsx:574]: The test description only covers the first use case (displaying the confirmation dialog). Splitting this test function could lead to better coherence and ensure the snapshot tests captures the modal a well

File: centreon/www/widgets/src/centreon-widget-resourcestable/src/Listing/translatedLabels.ts [hunk 3]
```diff
@@ -83,3 +85,9 @@ export const labelTicketOpenTime = 'Opened on';
 export const labelSubject = 'Subject';
 export const labelOpenTime = 'Open time';
 export const labelCreateticket = 'Create a ticket';
+export const labelCloseATicket = 'Close a ticket';
+export const labelConfirm = 'Confirm';
+export const labelTicketWillBeClosedInTheProvider =
+  'The ticket will be closed on the selected ticket provider';
```
Comment (cg-tw) [inline centreon/www/widgets/src/centreon-widget-resourcestable/src/Listing/translatedLabels.ts:91]:
```suggestion
  'The ticket will be closed for the selected ticket provider';
```

Comment (cg-tw) [reply centreon/www/widgets/src/centreon-widget-resourcestable/src/Listing/translatedLabels.ts:91]: I can't find the string elsewhere, is that normal?

Comment (Thebarda) [reply centreon/www/widgets/src/centreon-widget-resourcestable/src/Listing/translatedLabels.ts:91]: yes, can you add it in message.po please?

**Review by cg-tw — APPROVED** [review]

**Review by quborg — APPROVED** [review]

**Pull request merged**